openmls/src/treesync/treekem.rs

```diff
@@ -363,3 +361,72 @@ impl UpdatePath {
         self.nodes.push(last_node)
     }
 }
+
+#[derive(
+    Debug, PartialEq, Eq, Clone, Serialize, Deserialize, TlsSerialize, TlsDeserialize, TlsSize,
```

Comment:
This should not implement TlsSerialize, should it?

Reply:
This one is needed again in a -Tbs somewhere else